csrc/host_ir/jit.cpp

```diff
@@ -256,6 +419,7 @@ void compileFunctionDeclarations(llvm::Module* mod, llvm::LLVMContext& ctx) {
       kTensorSizeFuncName,
       mod);
 
+  // allocate_tensor function: at::Tensor* allocate_tensor()
   llvm::FunctionType* allocate_tensor_type =
       llvm::FunctionType::get(void_ptr_type, {}, false);
```

Comment:
For type safety, can this return `at::Tensor*`, with `at::Tensor` being an opaque type?